### PR TITLE
For Ember Times No. 132 - EmberConf Ad 🍓🥪

### PR DESCRIPTION
**Pull request by jayjayjpg**
Feel free to comment and suggest edits as you see fit @mansona!

### CODE REVIEW

File: source/2020-01-27-the-ember-times-issue-132.md [hunk 1]
```diff
@@ -56,13 +55,33 @@ If you have a great idea for a meetup, you can [sign up here](https://emberconf.
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Try your first Octane & JAM sandwich at EmberConf 🏎🥪🍓](https://emberconf.com/#/speakers/chris-manson)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Surely, you remember him as [the Emberista who built a blog site with Ember & NodeJs on stage in less than 15 mins](https://www.youtube.com/watch?v=GHbjdesHhPA), you heard him talk about [his work on an Ember-powered blog engine](https://www.youtube.com/watch?v=OnDUp2AhuNo) or you
```
Comment (ijlee2) [inline source/2020-01-27-the-ember-times-issue-132.md]: An optional suggestion. I thought using fewer syllables would deliver your punchline better.
```suggestion
Surely, you remember him as [the Emberista who built a blog site with Ember & Node in 15 min](https://www.youtube.com/watch?v=GHbjdesHhPA), you heard him talk about [his work on an Ember-powered blog engine](https://www.youtube.com/watch?v=OnDUp2AhuNo) or you
```

File: source/2020-01-27-the-ember-times-issue-132.md [hunk 2]
```diff
@@ -56,13 +55,33 @@ If you have a great idea for a meetup, you can [sign up here](https://emberconf.
...
+recall his [jammiest introduction to the JAM (JavaScript, APIs and Markup) stack for humans](https://www.youtube.com/watch?v=eQULGb10MbQ) with [46 mentions of the word "JAM" in total](https://twitter.com/real_ate/status/1207683401446313984)!
```
Comment (ijlee2) [inline source/2020-01-27-the-ember-times-issue-132.md]: I think we can switch the order to add more humor:
```suggestion
recall him [say JAM 46 times](https://twitter.com/real_ate/status/1207683401446313984) in his [jammiest introduction to the JAM (JavaScript, APIs and Markup) stack for humans](https://www.youtube.com/watch?v=eQULGb10MbQ)!
```

Comment (jayjayjpg) [reply source/2020-01-27-the-ember-times-issue-132.md]: Thank you, that sounds great!

File: source/2020-01-27-the-ember-times-issue-132.md [hunk 3]
```diff
@@ -56,13 +55,33 @@ If you have a great idea for a meetup, you can [sign up here](https://emberconf.
...
+This year, [Chris Manson (@mansona)](https://github.com/mansona) has even more insights into the modern JAM stack to share with you!
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+<!--alex disable king-queen-->
+At [this year's EmberConf](https://emberconf.com/) he's going to teach us about [**An Octane-Powered JAM Stack**](https://emberconf.com/#/schedule/day-1_an-octane-powered-jam-stack); how and why the JAM Stack has become so popular and how Ember Octane blends in well with this approach of web development. But that's not all: you can learn hands-on, how to build you first JAM stack site based on Octane and Empress in his EmberConf workshop [**Build and Publish Your Own empress-blog Template**](https://emberconf.com/#/schedule/pre-conf_build-and-publish-your-own-%60empress-blog%60-template).
```
Comment (ijlee2) [inline source/2020-01-27-the-ember-times-issue-132.md]: It's hard to see the diffs, sorry. I removed the word "how" from "how and why" (it seemed redundant), and edited the part "build you first ... in" to "build your first ... at".
```suggestion
At [this year's EmberConf](https://emberconf.com/) he's going to teach us about [**An Octane-Powered JAM Stack**](https://emberconf.com/#/schedule/day-1_an-octane-powered-jam-stack); why the JAM Stack has become so popular and how Ember Octane blends in well with this approach of web development. But that's not all: you can learn hands-on how to build your first JAM Stack site with Octane and Empress at his EmberConf workshop [**Build and Publish Your Own empress-blog Template**](https://emberconf.com/#/schedule/pre-conf_build-and-publish-your-own-%60empress-blog%60-template).
```

File: source/2020-01-27-the-ember-times-issue-132.md [hunk 4]
```diff
@@ -56,13 +55,33 @@ If you have a great idea for a meetup, you can [sign up here](https://emberconf.
...
+
+
+JAM Stack has become a popular phrase in our industry over the last few years with projects like Gatsby and Vuepress gaining popularity at fantastic speeds. As it is a very new tech stack there is still some innovation that can be added to the mix, and Ember Octane has the power to unlock a whole new generation of JAM Stack developers with the Empress project.
```
Comment (ijlee2) [inline source/2020-01-27-the-ember-times-issue-132.md]: I thought "popular idea" might have a more positive meaning than "popular phrase". The latter reminded me of something hyped unnecessarily.
```suggestion
JAM Stack has become a popular idea in our industry over the last few years. Projects like Gatsby and Vuepress are gaining popularity at fantastic speeds. As it is a new tech stack, there is still some innovation that can be added to the mix, and Ember Octane has the power to unlock a whole new generation of JAM Stack developers with the Empress project.
```

File: source/2020-01-27-the-ember-times-issue-132.md [hunk 5]
```diff
@@ -56,13 +55,33 @@ If you have a great idea for a meetup, you can [sign up here](https://emberconf.
...
+
+What will our experience building JAM stack websites with Octane be like? Chris shares from his experience as a JAM stack developer and as a workshop mentor:
+
+<!--alex ignore just-->
+> The thing that most people like when using JAM Stack with Ember, is the fact that you can get so much by just installing 1 or 2 addons. If you had to teach somebody everything from the ground all the way up to getting it deployed, it would take much longer and it would be much more difficult. But leveraging the power of Ember, leveraging the power of Ember addons makes people experience quick wins and gets a lot done very quickly.
+
+Besides from learning about the JAM stack in theory and in practice - what else makes EmberConf your go-to conference in 2020? Chris shares is perspective on what makes the event so worth being part of:
```
Comment (ijlee2) [inline source/2020-01-27-the-ember-times-issue-132.md]:
```suggestion
Besides learning about the theory and practice of JAM stack, what makes EmberConf your go-to conference in 2020? Chris shares his perspective on what makes the event so worth being part of:
```

File: source/2020-01-27-the-ember-times-issue-132.md (hunk 4 above)
Comment (jayjayjpg) [reply source/2020-01-27-the-ember-times-issue-132.md]: Woops, I copy-pasta'd the abstract from the schedule here for reference while writing the section and forgot to delete it again - will edit and thanks for the heads up!

**Review by ijlee2 — APPROVED** [review]
Thank you for covering another preview of an EmberConf talk and interview with the speaker!

**Comment by jayjayjpg** [comment]
🙇‍♀

**Pull request merged**